vars/defaults.yml

```diff
@@ -13,6 +13,8 @@ main_user_shell: "/bin/bash"
 # encfs_password: (required)
 friendly_networks:
   - ""
+# letsencrypt_email: (required)
```

Comment:
Providing an email address is [highly encouraged](https://community.letsencrypt.org/t/register-unsafely-without-email/4342).